packages/notifi-frontend-client/lib/client/ensureSource.ts

```diff
@@ -210,6 +210,22 @@ const ensureWalletBalanceSources = async (
           return 'ETHEREUM_WALLET';
         case 'BASE':
           return 'ETHEREUM_WALLET';
+        case 'BLAST':
+          return 'BLAST_WALLET';
+        case 'CELO':
+          return 'CELO_WALLET';
+        case 'INJECTIVE':
+          return 'INJECTIVE_WALLET';
```

Comment:
Seems `INJECTIVE_WALLET` is still not yet in `enum SourceType`. This will causes the typescript compile error

Reply:
I believe it is still not in prod since I cannot find `INJECTIVE_WALLET` any using prod playground (https://api.notifi.network/gql/)